src/audio/Kconfig

```diff
@@ -20,6 +20,13 @@ config COMP_DAI
 	help
 	  Select for DAI component
 
+config COMP_ARIA
+        bool "ARIA component"
+        default n
+        depends on IPC_MAJOR_4
+        help
+          Select for ARIA component
```

Comment:
Can you give a description of what ARIA does here and in the commit message. This will let users know when and how to use.